DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

```diff
@@ -19,9 +19,12 @@
 #include "CommonDataFormat/RangeReference.h"
 #include "CommonDataFormat/TimeStamp.h"
 #include "DataFormatsFT0/ChannelData.h"
+#include <FairLogger.h>
 #include <Rtypes.h>
 #include <gsl/span>
 #include <bitset>
+#include <iostream>
+#include "FairLogger.h"
```

Comment:
do you need iostream/FairLogger in the header? Better to avoid.

Reply:
removed now